packages/lending/src/protocol.ts

```diff
@@ -51,17 +51,15 @@ export abstract class Protocol extends common.Web3Toolkit {
 
   isAaveLike = false;
 
-  isUsingWrappedNativeToken = true;
-
   preferredFlashLoanerId?: string;
 
-  abstract newSupplyLogic(params: SupplyParams): Promise<SupplyLogic>;
+  abstract newSupplyLogic(params: any): api.Logic;
```

Comment:
Wondering why using `any` and `api.Logic` is a better option?

Reply:
@chouandy will fix after refactor

Reply:
This [commit](https://github.com/dinngo/protocolink-js-sdk/pull/15/commits/7834a8ac89cbcb65ac7f6ff2a915e54f1d36e60c#diff-b79db2de06cbe1ac07ca0c06c1ff25c20b3b0334c36d073027b973c17a8d802a) has fixed.